pallets/amendments/src/lib.rs

```diff
@@ -83,12 +82,12 @@ pub mod pallet {
 	impl<T: Config> Pallet<T> {
 		/// Schedule `amendment` to be executed after the configured time, unless vetoed by
 		/// `VetoOrigin`
-		#[pallet::weight(
-			(
-				T::WeightInfo::propose(
-					amendment.using_encoded(|x| x.len()) as u32,
-				).saturating_add(amendment.get_dispatch_info().weight),
-				DispatchClass::Operational,
+		#[pallet::weight((
+			T::WeightInfo::propose(
+				amendment
+					.using_encoded(|x| x.len()) as u32)
+					.saturating_add(amendment.get_dispatch_info().weight),
+					DispatchClass::Operational,
```

Comment:
Can you please explain where this change has come from and how it's related to to without_storage_info?

Reply:
looks like from `cargo fmt`, let me recheck this

Reply:
This doesn't look just formatting to me. The result of `propose(amendment)+amendment.weight` can be different from `propose(amendment + amendment.weight)`.

Reply:
You are right, Done